src/components/hv-view/index.js

```diff
@@ -40,6 +40,53 @@ export default class HvView extends PureComponent<HvComponentProps> {
 
   props: HvComponentProps;
 
+  isHiddenElement = (attributes: Attribute[]): boolean => {
```

Comment:
curious, why are you passing the attributes rather than the entire `Element`? Then you can do `return element.getAttribute("name") === "true"`

Reply:
Same reason as below. Makes sense, updating to pass an element instead.